gradle/dependencies.gradle

```diff
@@ -23,7 +23,7 @@ ext {
       mapboxEvents              : '8.1.2',
       mapboxCore                : '5.0.1',
       mapboxNavigator           : "${mapboxNavigatorVersion}",
-      mapboxCommonNative        : '22.0.0-rc.2',
+      mapboxCommonNative        : '22.0.0',
```

Comment:
should also be updated to the new map SDK with common 22 onboard 🤔 https://github.com/mapbox/mapbox-maps-android/releases/tag/android-v10.6.0